app/views/devise/registrations/_new_family.html.erb

```diff
@@ -27,21 +27,73 @@
                         label: "Confirmez le mot de passe",
                         required: true,
                         input_html: { autocomplete: "new-password", class: "seko-search-bar" } %>
-            <%= f.input :address,
-                        label: "Adresse",
+
+            <%= f.input :link,
+                        label: "Quel est votre lien avec le bénéficiaire?",
                         required: true,
-                        input_html: { autocomplete: "address", :id => "family_address", class: "seko-search-bar" }  %>
-            <%= f.input :address,
-                        label: "Complément d'adresse",
+                        collection: [
+                          ['Parent proche', 'Parent Proche'],
+                          ['Ami(e)', 'Ami(e)'],
```

Comment:
Carefull, in a collection, in each array, the first element will be the value to store in the DB and the second will be the text display to the user. So no spaces or upcase for the first element.
["close_parent", "Parent Proche"] or ["0", "Parent Proche"]
Same for the second array with "Amie".

Reply:
I've updated arrays